bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/FindReplaceLogic.java

```diff
@@ -141,7 +141,8 @@ public boolean isRegExSearchAvailableAndActive() {
 	 * Initializes the anchor used as starting point for incremental searching.
 	 *
 	 */
-	private void initIncrementalBaseLocation() {
+	@Override
+	public void resetIncrementalBaseLocation() {
```

Comment:
The comment of this method is obsolete, now that it is implementing an already documented interface method.

Reply:
Removed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/f284814f4e9a8c2034555608d15c4837b0841875